bumble/device.py

```diff
@@ -1301,6 +1308,10 @@ async def power_on(self) -> None:
                 )
 
         if self.classic_enabled:
+            # setup pin type for legacy pairing
+            await self.send_command(
```

Comment:
This command is optional, I think we should check if it's supported before sending it.